src/coreclr/debug/daccess/daccess.cpp

```diff
@@ -5730,6 +5730,87 @@ ClrDataAccess::GetJitHelperName(
     return NULL;
 }
 
+// This function expects more memory than maybe needed.
+static int FormatCLRStubName(
+    _In_opt_z_ LPCWSTR stubNameMaybe,
+    _In_ TADDR stubAddr,
+    _In_ ULONG32 bufLen,
+    _Out_ ULONG32 *symbolLen,
+    _Out_writes_bytes_opt_(bufLen) WCHAR* symbolBuf)
+{
+    // Parts needed to construct a name:
+    // With stub manager name: "CLRStub[%s]@%p"
+    //   No stub manager name: "CLRStub@%p"
+    const WCHAR formatName_Prefix[] = W("CLRStub");
+    const WCHAR formatName_OpenBracket[] = W("[");
+    const WCHAR formatName_CloseBracket[] = W("]");
+    const WCHAR formatName_PrefixEnd[] = W("@");
+    const size_t formatName_AddrMaxLen = ARRAY_SIZE("ffffffffffffffff") - 1; // 64-bit
```

Comment:
It is hard to tell that this string is correct by just looking at it. If this string had 15 characters instead of 16, nobody will notice that during codereview.